src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Map/AccessLoadStatus/AccessLoadStatus.xaml

```diff
@@ -8,15 +8,14 @@
     mc:Ignorable="d"
     d:DesignHeight="300"
     d:DesignWidth="400">
-
     <Grid>
-        <Grid.RowDefinitions>
-            <RowDefinition Height="auto" />
-            <RowDefinition Height="*" />
-        </Grid.RowDefinitions>
-        <StackPanel Orientation="Horizontal">
-            <TextBlock x:Name="loadStatusTextBlock" Text="Map's load status : "/>
-        </StackPanel>
-        <esriUI:MapView x:Name="myMapView"  Grid.Row="1"/>
+        <esriUI:MapView x:Name="myMapView" />
```

Comment:
More inconsistently-capitalized XAML fields: `myMapView` and `loadStatusTextBlock`